src/pydata_sphinx_theme/theme/pydata_sphinx_theme/theme.conf

```diff
@@ -28,9 +28,11 @@ show_nav_level = 1
 show_toc_level = 1
 navbar_align = content
 navbar_start = navbar-logo.html
-navbar_center = navbar-nav.html
+navbar_center = theme-switcher.html, navbar-nav.html
```

Comment:
WDYT about putting it in the navbar_end instead?
IMHO, it feels "displaced" where it is currently at.